### PR TITLE
dynamic metadata

### PR DESCRIPTION
**Pull request by michielkoning**
Dynamic metadata based on vue-meta. I wasn't sure if the metaData file should be a helper, a utility, a hook.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,25 +5,14 @@
...
     <!-- Open Graph data -->
-    <meta property="og:title" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests" />
-    <meta property="og:site_name" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests" />
-    <meta property="og:url" content="http://police-accountability.netlify.app/" />
-    <meta property="og:image" content="http://police-accountability.netlify.app/police-seo.png" />
-    <meta property="og:description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable." />
-    <meta property="og:image:width" content="602" />
-    <meta property="og:image:height" content="602" />
-    <link href="https://fonts.googleapis.com/css2?family=Bungee&family=Open+Sans&display=swap" rel="stylesheet">
+    <meta property="og:site_name" content="%=VUE_APP_TITLE %>" />
```
Comment (dmnd) [inline public/index.html:14]: 🐛

```suggestion
    <meta property="og:site_name" content="<%=VUE_APP_TITLE %>" />
```

File: public/index.html [hunk 2]
```diff
@@ -5,25 +5,14 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>Police Abuses: Hundreds of Videos of Police Brutality from George Floyd Protests</title>
-    <meta name="description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable."/>
+    <title><%=VUE_APP_TITLE %></title>
     <!-- Twitter Card data -->
     <meta name="twitter:card" content="summary">
-    <meta name="twitter:site" content="http://policeabuses.com">
-    <meta name="twitter:title" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests">
-    <meta name="twitter:description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable.">
```
Comment (dmnd) [inline public/index.html:14]: This is the only description that will work without server-side rendering so might be better to keep it around.

File: public/index.html [hunk 3]
```diff
@@ -5,25 +5,14 @@
...
-    <meta name="twitter:image" content="http://police-accountability.netlify.app/police-twitter.png">
-    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="<%=VUE_APP_BASE_URL %>">
 
     <!-- Open Graph data -->
-    <meta property="og:title" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests" />
-    <meta property="og:site_name" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests" />
-    <meta property="og:url" content="http://police-accountability.netlify.app/" />
-    <meta property="og:image" content="http://police-accountability.netlify.app/police-seo.png" />
-    <meta property="og:description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable." />
```
Comment (dmnd) [inline public/index.html:23]: Same here

File: public/index.html (hunk 2 above)
Comment (michielkoning) [reply public/index.html:14]: I don't understand your feedback. These attributes are being set based on the incidents once vue kicks it.

**Pull request merged**